src/index.js

```diff
@@ -0,0 +1,405 @@
+
+const ticketsModalBasik = document.querySelector('.quantinyOfBasic')
+const ticketsModalSenior = document.querySelector('.quantinyOfSenior')
+
+const totalSumOfBasikModal = document.querySelector('.totalSumOfBasic')
+const totalSumOfSeniorModal = document.querySelector('.totalSumOfSenior')
+
+const basicPriceFirst = document.querySelector('.basicPriceFirst')
+const seniorPriceFirst = document.querySelector('.seniorPriceFirst')
+const basicPriceSecond = document.querySelector('.SumOfBasic')
+const seniorPriceSecond = document.querySelector('.SumOfSenior')
+
+const createStyleFlex = document.querySelector('.inputItems__entryTicket__value2')
+
+const blockRadioBtns = document.querySelector('.tickets__inf__calc__items')
+
+let sum = 30;
+function radioHelper(item,priceBasik, priceSenior ){
+  sum = (parseInt(countInput.value) * priceBasik) + (parseInt(countInput2.value) * priceSenior)
+  totalSum.innerHTML = `Total € ${sum}`
+  totalModalSum.innerHTML = sum;
+  selectTicket.value = item.value
+  valueTicketsSpan.innerHTML = item.innerHTML
+  valueTicketsSpan.innerHTML = item.innerHTML
+  totalSumOfBasikModal.innerHTML =  priceBasik * parseInt(countInput.value)
+  totalSumOfSeniorModal.innerHTML = priceSenior * parseInt(countInput2.value)
+
+  basicPriceFirst.innerHTML = priceBasik
+  seniorPriceFirst.innerHTML = priceSenior
+
+  basicPriceSecond.innerHTML = priceBasik
+  seniorPriceSecond.innerHTML = priceSenior
+
+  createStyleFlex.classList.remove('testGap')
+}
+function testRadio (arg) {
+  sum = 0
+  if((arg === 'first') || (arg === 'firstT')){
+   radioHelper(options[0],priceFirstTicketBasic,priceFirstTicketSenior)
+  }
+  else if((arg === 'second') || (arg === 'secondT')){
+    radioHelper(options[1],priceSecondTicketBasic,priceSecondTicketSenior)
+  }
+  else if ((arg === 'third') || (arg === 'thirdT')){
+    radioHelper(options[2],priceThirdTicketBasic,priceThirdTicketSenior)
+  }
+ }
+
+let priceFirstTicketBasic = 20;
+let priceFirstTicketSenior = 10;
+
+let priceSecondTicketBasic = 25;
+let priceSecondTicketSenior = 12.5;
+
+let priceThirdTicketBasic = 40;
+let priceThirdTicketSenior = 20;
+
+ blockRadioBtns.addEventListener('click',(event) =>{
+      let target = event.target
+      if(target === radioBtns[0]){ 
+        testRadio(event.target.value)
+      } else if(target === radioBtns[1]){
+        testRadio(event.target.value)
+      } else if (target === radioBtns[2]){
+        testRadio(event.target.value)
+      }
+    })
+
+const wrapper = document.querySelector('wr');
+let countInput = document.getElementById('count');
+let butM = document.getElementById('bminus');
+let butP = document.getElementById('bplus');
+let units = countInput.value.replace(/\d/g, '');
+butP.addEventListener('click', function(){
+   
+  countInput.value = parseInt(countInput.value)+1+units;
+  ticketsModalBasik.innerHTML = countInput.value
+  countInput3.value = countInput.value;
+  if(radioBtns[0].checked === true){
```

Comment:
Не надо проверять напрямую === true

Это можно проверить так

```suggestion
  if(radioBtns[0].checked){
```